### ISSUE TITLE
Running TPS without known receptors

### ISSUE THREAD

**Issue by agitter** [koksal/tps#13]
In https://github.com/koksal/tps/issues/10#issuecomment-752196974 @ozgunbabur asked
> How can we use TPS to model proteomic differences when we don't know the exact location of the perturbation on the network? For instance, a cell type is treated with two different types of media (e.g. bovine serum vs PBS). In that case, does it still make sense to use PCST? Do you have any best practice for such cases?

I wanted to open a new issue to discuss this.  There could be multiple approaches, and we may want to keep this discussion active even after #10 is resolved.

My first thought would be to use a large and general list of source proteins or receptors as the source nodes for both PCSF and TPS.  I don't have experience using TPS in this manner.  However, a similar approach was used with PCSF in [Tuncbag et al. 2013](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3576906/).  There they used receptors from the Human Plasma Membrane Database.  My hope is that when PCSF selects a subnetwork, it will prune the receptors to a smaller set that is reasonable for TPS.

An alternative idea would be to look at your proteomic data and see which receptors are differentially phosphorylated.  Those could be used as the PCSF and TPS sources.

I'm interested in supporting this use case and can comment on any preliminary results you get from PCSF or TPS with suggestions.
